src/coreclr/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -181,6 +181,16 @@ target_compile_definitions(coreclr_static PUBLIC CORECLR_EMBEDDED)
 if(CLR_CMAKE_TARGET_WIN32)
     set(CLRDEBUGINFO_RESOURCE_PATH ${CMAKE_CURRENT_BINARY_DIR}/clr_debug_resource.bin)
 
+    if(CLR_CMAKE_TARGET_ARCH_AMD64)
+        set(CLRDEBUGINFO_RESOURCE_NAME CLRDEBUGINFOWINDOWSAMD64)
```

Comment:
Were we not emitting it for any arch?

Reply:
dbgshim only looks at these architectures and I don't think on Windows we support anything other than amd64, arm64, arm and x86.

Reply:
Meant more as to when this was changed. I don't see any version of CoreCLR or CLR that has the arch specific lookup. I do see the code in DBGShim what reads it. I just don't know how it worked or if it was used by any other component (like FX).

Reply:
As far as I know, this resource is only used by dbgshim.  clrmd uses "CLRDEBUGINFO" only.